### ISSUE TITLE
Build Contrast Utility

### ISSUE THREAD

**Issue by codewithkyle** [Pageworks/papertrain#10]
We should have our font colours be lighter based on the background color and the contrast ratio. Look into building a mixin to handle this [example](https://medium.com/dev-channel/using-sass-to-automatically-pick-text-colors-4ba7645d2796).
